### DISCUSSION TITLE
Unable to start - failed to create shim task: OCI runtime create failed: runc

### DISCUSSION THREAD

**Discussion by DSew20** [photoprism/photoprism#2426] [Q&A]
Hello,

I had PhotoPrism working, but alas it has stopped.  Not sure what has happened, but I now get the following error:

Failure
failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/entrypoint.sh": stat /entrypoint.sh: no such file or directory: unknown

I'm running PhotoPrism in docker, managed via Portainer.  Was working ok, then tried to open today and I get the above message.

Any ideas where to start to decode / debug this?  MariaDB seems to start ok.

**Comment by lastzero** [comment]
Check the environment variables as Portainer tends to cache those of old versions and applies them to new images as well even though they do not fit and can break things completely. Search our issues and discussions for the reports of other Portainer users.

**Reply by DSew20** [reply]
Thank you.  I can't see any Env values that have changed.  I searched the forum, issues and discussions but nothing jumps out as a similar issue.

Any further pointers?  Thank you.  I'm keen to get PhotoPrism working as I spent a long time sorting out my photos but it is not working now and not sure why it has stopped.

**Reply by DSew20** [reply]
I see in #2120 there is reference to removing PATHS from the Env.

I have removed all 6 *PATH variables, but alas still does not start.

**Reply by lastzero** [reply]
Might also be related:
- https://github.com/photoprism/photoprism/issues/1301

**Reply by lastzero** [reply]
If you need further assistance, please share your logs and anything else that might be useful to give more specific advice! 👍

**Reply by DSew20** [reply]
Thank you.  I think using the prior version helped, but now I am getting a different error.  Duplicate Entry!?!

`running as uid 0
photoprism start
time="2022-06-15T20:55:13Z" level=debug msg="entity: migrate Error 1062: Duplicate entry 'Innsbruck, Tirol, Austria' for key 'uix_places_place_label' (waiting 1s)"
panic: Error 1062: Duplicate entry 'Innsbruck, Tirol, Austria' for key 'uix_places_place_label'

goroutine 1 [running]:
github.com/photoprism/photoprism/internal/entity.Types.Migrate(0xc00031de30)
	/go/src/github.com/photoprism/photoprism/internal/entity/entity.go:107 +0x22f
github.com/photoprism/photoprism/internal/entity.MigrateDb()
	/go/src/github.com/photoprism/photoprism/internal/entity/entity.go:135 +0x2d
github.com/photoprism/photoprism/internal/config.(*Config).InitDb(0xc0002d64b0)
	/go/src/github.com/photoprism/photoprism/internal/config/db.go:226 +0x49
github.com/photoprism/photoprism/internal/commands.startAction(0xc0000ed760, 0xc000331800, 0xc0003318a0)
	/go/src/github.com/photoprism/photoprism/internal/commands/start.go:75 +0x216
github.com/urfave/cli.HandleAction(0xf63c00, 0x11400d0, 0xc0000ed760, 0xc0000ed760, 0x0)
	/go/pkg/mod/github.com/urfave/cli@v1.22.5/app.go:524 +0x105
github.com/urfave/cli.Command.Run(0x10b7e30, 0x5, 0x0, 0x0, 0x187e900, 0x1, 0x1, 0x10d1b3b, 0x11, 0x0, ...)
	/go/pkg/mod/github.com/urfave/cli@v1.22.5/command.go:173 +0x579
github.com/urfave/cli.(*App).Run(0xc000524700, 0xc0000ac000, 0x2, 0x2, 0x0, 0x0)
	/go/pkg/mod/github.com/urfave/cli@v1.22.5/app.go:277 +0x808
main.main()
	/go/src/github.com/photoprism/photoprism/cmd/photoprism/photoprism.go:75 +0x629`

**Reply by lastzero** [reply]
Maybe some migrations didn't run properly. Running this command in a terminal (if using our examples with docker-compose) will show the status of existing migrations:

```bash
docker-compose exec photoprism photoprism migrations ls 
```

**Reply by lastzero** [reply]
Check if running the container as root works if you currently have any special user settings.

**Reply by DSew20** [reply]
> 

Thank you.  A newb question (sorry) - how do run the command if using Portainer?  I don't want to mess up other containers as everything else is running ok.

**Reply by lastzero** [reply]
Check their docs :) We are not actively using Portainer... however, we should cover it better in our docs, just didn't have a second of time yet.

**Comment by DSew20** [comment]
Thank you again lastzero

anybody - 
I have tried every combination I can think of and checked the Portainer docs to run the below command in Portainer:

`docker-compose exec photoprism photoprism migrations ls `

I've tried in the Advanced Container settings and the Console.  But, I cant get the above to run / Portainer / Docker does not parse the command.

Can anybody suggest how to run exec migration ls from Portainer?
